newsletters/007.md

```diff
@@ -2,10 +2,18 @@
 
 ## Editor's Note
 
-Welcome to the seventh issue of the SG2042 Newsletter. 
+Welcome to the seventh issue of the SG2042 Newsletter. We are excited to bring you the great news that SOPHON SG2042 was presented at the top international academic conference in the field of high performance computing!
 
 ## Highlights
 
++ Based on the SOPHON SG2042, more and more researchers are able to explore high-performance RISC-V and publish their research results.
+
+  [Link to the paper](https://arxiv.org/pdf/2309.00381.pdf)
+
++ Milk-V Duo development board has been compatible with over 50 peripherals by now, including ST7789, NFC PN532, PAJ7620 gesture sensor, OLED12864, ENS160 air quality sensor, MAX30102 heart rate and SpO2 sensor, etc.

```

Comment:
`has been compatible with`
=>
`is now compatible with`

Reply:
Thank you for your advice. It has been revised.